examples/proposed/cacheable-function.w

```diff
@@ -9,7 +9,7 @@ resource CachableFunction extends cloud.Function {
         this._cache = new cloud.Bucket();
     }
 
-    public override inflight invoke(event: any) {
+    public override inflight invoke(event: any): any {
```

Comment:
I think we don't have `any` in wing, but I guess it looks better than those `: str` return types we have everywhere, so maybe just keep it as is (just `bring`ing this point to attention).